Source/WTF/wtf/CheckedRef.h

```diff
@@ -196,6 +196,18 @@ inline bool is(const CheckedRef<ArgType, ArgPtrTraits>& source)
     return is<ExpectedType>(source.get());
 }
 
+template<typename ExpectedType, typename ArgType, typename ArgPtrTraits>
+inline ExpectedType& downcast(CheckedRef<ArgType, ArgPtrTraits>& source)
+{
+    return downcast<ExpectedType>(source.get());
+}
+
+template<typename ExpectedType, typename ArgType, typename ArgPtrTraits>
+inline ExpectedType& downcast(const CheckedRef<ArgType, ArgPtrTraits>& source)
```

Comment:
Seems this should return a `const ExpectedType&` no? Since CheckedRef::get() returns a `const T&` when called on a const CheckedRef.